changelog.d/20241015_111820_max.tuecke_sc_35029_deprecate_delete_flag.md

```diff
@@ -0,0 +1,5 @@
+Added
+~~~~~~~
```

Comment:
In restructured text it's expected the markup for the header is the same number of characters
```suggestion
Added
~~~~~
```